examples/manifests/kustomization.yaml

```diff
@@ -14,5 +14,5 @@ images:
   newName: quay.io/redhat-cne/cloud-event-consumer
   newTag: latest
 - name: cloud-event-sidecar
-  newName: quay.io/redhat-cne/cloud-event-proxy
+  newName: quay.io/aneeshkp/origin-cloud-event-proxy
```

Comment:
I think this needs to point to the redhat-cne version

Reply:
Done